src/System.Text.Primitives/System/Text/FormattingData.cs

```diff
@@ -142,6 +503,9 @@ public enum Symbol : ushort
             PlusSign = 14,          
             NaN = 15,
             Exponent = 16,
+            ExponentSecondary = 17,
```

Comment:
Can you add a comment describing what this is? I think he other names come from the Unicode standard. I don't remember seeing "secondary exponent" there, so people might not be able to understand what this is.